predict.py

```diff
@@ -0,0 +1,91 @@
+"""
+To push this predictor to replicate.com, first run download_checkpoints() and save files to omnizart/checkpoints.
+Then run cog push. Further documentation can be found at https://replicate.com/docs
+"""
+
+import os
+import tempfile
+from pathlib import Path
+import subprocess
+import shutil
+import cog
+import scipy.io.wavfile as wave
+from omnizart.music import app as mapp
+from omnizart.chord import app as capp
+from omnizart.drum import app as dapp
+from omnizart.vocal import app as vapp
+from omnizart.vocal_contour import app as vcapp
+from omnizart.beat import app as bapp
+from omnizart.remote import download_large_file_from_google_drive
+
+
+class Predictor(cog.Predictor):
+    def setup(self):
+        self.SF2_FILE = "general_soundfont.sf2"
+        if not os.path.exists(self.SF2_FILE):
+            print("Downloading soundfont...")
+            download_large_file_from_google_drive(
+                "16RM-dWKcNtjpBoo7DFSONpplPEg5ruvO", file_length=31277462, save_name=self.SF2_FILE
+            )
+        self.app = {"music": mapp, "chord": capp, "drum": dapp, "vocal": vapp, "vocal-contour": vcapp, "beat": bapp}
+        self.model_path = {"piano": "Piano", "piano-v2": "PianoV2", "assemble": "Stream", "pop-song": "Pop", "": None}
+
+    @cog.input(
+        "audio",
+        type=Path,
+        help="input polyphonic music mp3 file",
+    )
+    @cog.input(
+        "mode",
+        type=str,
+        default="music-piano-v2",
+        options=["music-piano", "music-piano-v2", "music-assemble", "chord", "drum", "vocal", "vocal-contour", "beat"],
+        help="transcribe mode",
+    )
+    def predict(self, audio, mode):
+        assert str(audio).rsplit('.', maxsplit=1)[-1] == "mp3", "supports mp3 file"
```

Comment:
Two things to be mentioned.
First, `omnizart` accepts `.wav` files. It seems like the code here are modified from the colab. The reason why colab suggests upload mp3 files is because the upload speed is pretty slow, which takes a pretty long time to upload `.wav` files. So if you want to check the format here, you should also accept `.wav` format.

Second, you could write `str(audio).endswith(".mp3")`, and that is enough. And also the error message should be more verbose. The message can not express clear enough what the actual error is.